src/code/providers/interactive-api-provider.ts

```diff
@@ -13,12 +13,43 @@ import {
 } from '@concord-consortium/lara-interactive-api'
 import { SelectInteractiveStateDialogProps } from '../views/select-interactive-state-dialog-view'
 
+export const shouldSaveAsAttachment = (content: any) => {
+  const interactiveApi = queryString.parse(location.search).interactiveApi
+  if (interactiveApi === kAttachmentUrlParameter) {
+    return true
+  }
+
+  const aboveDynamicThreshold = JSON.stringify(content).length >= kDynamicAttachmentSizeThreshold
+  if (aboveDynamicThreshold) {
+    return true
+  }
+
+  return false
```

Comment:
Could just:
```
  return aboveDynamicThreshold
```

Reply:
Yep - I just copied that function without change out of the class so it could be used outside of it.